app/views/admin/statistics_announcements/show/_main.html.erb

```diff
@@ -0,0 +1,137 @@
+<div class="app-view-statistics-announcements-summary__main">
+  <section class="app-view-statistics-announcements-summary__section page-header">
+    <p class="govuk-body-lead">
+      <%= @statistics_announcement.summary %>
+    </p>
+
+    <div>
+      <%=
+        render "govuk_publishing_components/components/summary_list", {
+          borderless: true,
+          items: [
+            { field: "Type", value: @statistics_announcement.display_type },
+            { field: "Organisations", value: @statistics_announcement.organisations.map(&:name).to_sentence },
+          ],
+        }
+      %>
+    </div>
+
+    <div>
+      <%= render "components/inset_prompt", {
+        error: true,
+        description: "Statistics release cancelled - #{@statistics_announcement.cancellation_reason}.",
+      } if @statistics_announcement.cancelled? %>
+    </div>
+  </section>
+
+  <section class="app-view-statistics-announcements-summary__section">
+    <div>
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Release and delivery date",
+        heading_level: 2,
+        font_size: "l",
+        margin_bottom: 3,
+      } %>
+      <p class="govuk-body">
+        <%= link_to("Change dates",
+                    new_admin_statistics_announcement_change_path(@statistics_announcement),
+                    class: "govuk-link",
+                    data: {
+                      module: "gem-track-click",
+                      "track-category": "button-clicked",
+                      "track-action": "#{@statistics_announcement.model_name.singular.dasherize}-button",
+                      "track-label": "Change dates",
+                    }) %>
+      </p>
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Release date",
+        heading_level: 3,
+        font_size: "m",
+        margin_bottom: 2,
+      } %>
+      <% if @statistics_announcement.cancelled? %>
+       <p>
+         <del class="govuk-body"><%= @statistics_announcement.release_date.to_fs(:date_with_time) %></del>
```

Comment:
Just checking we want to use a strike through here? I've not seen used on a service before

Reply:
This is replicating the behaviour on the legacy. So maybe one for @nnagewad to opine on. 

@nnagewad do we want the release date to have strike through when cancelled as per the legacy implementation?